packages/adblocker/src/filters/network.ts

```diff
@@ -151,6 +151,7 @@ export const enum NETWORK_FILTER_MASK {
   isHostnameAnchor = 1 << 28,
   isRedirectRule = 1 << 29,
   isRedirect = 1 << 30,
+  isRemoveParam = 1 << 31,
```

Comment:
To confirm, we're fine with negative numbers as mask?

Reply:
Yes, it is addressed by #4634 and integrity tests.